SalesforceCommerceCloud/pwa-kit
# [App Extensibility ⚙️] Extensibility Preview Release (@W-17273312@)

# Description



This PR bumps the initial extensibility preview release version for all packages in the monorepo.

- Core packages are released with the version `4.0.0-extensibility-preview.0`.
- New extension packages are released with the version `0.1.0-extensibility-preview.0`.




# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Added new extensions packages to Bump-version script.

# How to Test-Drive This PR

- Verify the `version` in package.json is using the expected versions.
- Generate projects using the different generator flows and verify things work as expected. 

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
